testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeViewComments.java

```diff
@@ -0,0 +1,56 @@
+package io.trino.tests.product.deltalake;
+
+import org.testng.annotations.Test;
+
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_DATABRICKS;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_OSS;
+import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestDeltaLakeViewComments
```

Comment:
I haven't found an existing class suitable for this test. Furthermore, I have not found any product testing code for comment on view (not on view column) in `io.trino.tests.product.deltalake`. If it does not exist, perhaps we should also add one?

Reply:
Sure, It doesn't matter. I have dropped this commit. 😆